### DISCUSSION TITLE
Potential Revision

### DISCUSSION THREAD

**Discussion by TriumphReturns** [misternoe/cheatbox#10] [Q&A]
I currently have a version of the D-Brake for a GamerFinger button which uses Cherry MX Switches was wondering if it was possible to get one that has the inner of the button filled out and not hollow like the Sanwa/Seimitsu model. It is currently very flimsy. Would love it in a 7mm height as well.

**Comment by misternoe** [comment]
So this is something I planned to explore once my 24mm gamerfinger button arrives. I took an identical approach when creating the prototype of the D-BRAKE for crown buttons and it does help with stability. 

That said, adding a cover will potentially affect compatibility with gamerfingers and low-profile cherry mx buttons, like the punk workshop buttons. So more than likely there would need to be a version for each button.

As for a 7mm version, i don't know if that will be possible with the 8mm prototype already making contact with the button rim according to a tester.

**Comment by TriumphReturns** [comment]
Ahh I see. Glad to know you're already working on it! I'll be keeping a
look out for it then. Whatever the lowest you can get it is fine with me.
Also for a 30mm Gamerfinger as well.
